cluster-autoscaler/processors/emptycluster/empty_cluster_processor.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package emptycluster
+
+import (
+	"time"
+
+	apiv1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/cluster-autoscaler/clusterstate/utils"
+	"k8s.io/autoscaler/cluster-autoscaler/context"
+	"k8s.io/autoscaler/cluster-autoscaler/metrics"
+	"k8s.io/autoscaler/cluster-autoscaler/utils/errors"
+	"k8s.io/klog/v2"
+)
+
+//NodesNotReadyAfterStartTimeout How long should Cluster Autoscaler wait for nodes to become ready after start.
+const NodesNotReadyAfterStartTimeout = 10 * time.Minute
+
+// EmptyClusterProcessor defines interface for whether action needs to be taken for the a cluster or not
+type EmptyClusterProcessor interface {
+	// ActOnCluster is the func that will return where the cluster is in an actionable state or not
+	// It should also help with extensibility by allowing custom actionable conditions by different implementations
```

Comment:
I think that sentence is why we have any processor at all, so it seems redundant.

Reply:
removed